pkg/features/kube_features.go

```diff
@@ -58,6 +58,16 @@ const (
 	// Enables usage of any object for volume data source in PVCs
 	AnyVolumeDataSource featuregate.Feature = "AnyVolumeDataSource"
 
+	// owner: @MikeSpreitzer, @tkashem, @linxiulei
+	// beta: 1.33
```

Comment:
nit: we no longer put stage in the comments (it's reversible from the versioned feature list)

Reply:
removed.